src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -837,38 +837,40 @@ public async Task EnableDatabases()
             await ToggleDisableDatabases(disable: false);
         }
 
+        [RavenAction("/admin/databases/toggle-indexing", "POST", AuthorizationStatus.Operator)]
```

Comment:
I do not like the inconsistency between this endpoint and `/admin/databases/dynamic-node-distribution` in naming

Reply:
done